src/components/Stepper.jsx

```diff
@@ -0,0 +1,59 @@
+import { React, useState } from "react";
+
+import Button from "./ui/Button";
+
+function Stepper(props) {
+    const { steps, currentStep } = props;
+    const [current, setCurrent] = useState(currentStep);
+    const nextStep = () => {
+        if (current < steps.length - 1) {
+            setCurrent(current + 1);
+        }
+    };
+
+    const prevStep = () => {
+        if (current > 0) {
+            setCurrent(current - 1);
+        }
+    };
+
+    return (
+        <div className='flex flex-col items-center justify-center'>
+            <div className='max-w-sm p-6 border-gray rounded-lg shadow dark:bg-gray-800 dark:border-gray-700'>
+                {steps.map((step, index) => {
+                    return (
+                        <div
+                            key={index}
+                            className={`${
+                                current === index ? "block" : "hidden"
+                            } w-full h-full p-4 text-center`}
+                        >
+                            <h1 className='text-4xl font-bold'>{step.title}</h1>
```

Comment:
As mentioned, with regards to our theme, (text-4xl, font-bold) is not choices in our theme, we have only (light, normal, medium, semibold for fontWeight, and sm, lg, xl, 2xl, ...etc). but if you want to write another value that doesnt exit there, you should use [ ], to be font-[bold], text-[4xl]